src/screens/Home/CreateScreen/VideoLoadingScreen.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import AnimatedEllipsis from 'react-native-animated-ellipsis';
+import {
+  Text,
+  View,
+} from 'react-native'
+import styles from './styles'
+
+export default function CreateScreen() {
+  return (
+    <View style={styles.container}>
+        <View style={styles.main}>
+            <AnimatedEllipsis numberOfDots={4}
+                  minOpacity={0.5}
+                  animationDelay={200}
+                  style={styles.ellipses}
+            />
+            <Text style={styles.text}>CREATING YOUR VIDEO</Text>
+        </View>
+    </View>
+  )
+}
+
```

Comment:
The spacing on this file and a few others looks a bit off.
We have a standardized linter that enforces a common styling across all our code. Makes it easier to read for everyone and makes our code more professional.

A simple way to run ours is in terminal, `yarn lint --fix`. It will fix all of the spacing issues. You may be left with some linter warnings about unused variables - you can ignore them.